ui/src/app/views/homepage/component.css

```diff
@@ -1,251 +1,221 @@
-.front-page {
-    margin-left: -1rem;
-    margin-right: -.6rem;
-    text-align: center;
-    position: relative;
-}
-
 .background-image {
-    background-image: url('/assets/images/homepage.png');
+    background-image: url('/assets/images/AoU-HP-background.jpg');
     background-repeat: no-repeat;
-    z-index: -1;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
     background-size: cover;
-    /*height: calc(100vh - 3.3rem);*/
-    height: 41rem;
-}
-
-.rectangle {
-    opacity: .47;
-    background-color: #000000;
-    box-shadow: 0 1px 4px 0 rgba(0,0,0,0.28);
-    position: absolute;
-    margin-top: 5rem;
-    left: 50%;
-    transform: translate(-50%, 0);
-    width: 80%;
-    height: 11rem;
-    z-index: 1;
-}
-
-.invisible_rectangle {
-    margin-top: 5rem;
-    width: 80%;
-    z-index: 2;
-    height: 11rem;
-    position: absolute;
-    left: 50%;
-    transform: translate(-50%, 0);
-}
-
-.tip-title {
-    color: #FFFFFF;
-    padding-top: 1rem;
-    padding-left: 2rem;
-    font-size: 1.8rem;
-    font-weight: 300;
-    line-height: 2rem;
-    position: absolute;
+    height:100%
 }
 
-.tip-subtitle {
-    padding-top: 3.7rem;
-    padding-left: 2.1rem;
-    font-size: .9rem;
-    line-height: 1rem;
+.card {
+    border-radius: 5px;
```

Comment:
We should have a border radius for cards in the card master style file. Does this need to be here/is this different than that??